src/utils/localFormatted/localFormatted.spec.ts

```diff
@@ -1,27 +1,19 @@
-import {formattedDate, formattedNumber} from './localFormatted';
+import {formattedNumber} from './localFormatted';
 
+const locale = 'en-US';
 const results = {
-  'en-US': {
-    expectNumber: '50,000.05',
-    expectNumberWithAccuracy: '50,000.0500',
-    expectDate: '12/17/1995'
+  [locale]: {
+    expectNumber: '50,000.050',
+    expectNumberWithAccuracy: '50,000.050'
   }
 };
 
 describe('Test locale for number and date', () => {
-  const date = new Date(Date.UTC(1995, 11, 17, 3, 24, 0));
-  const locale = navigator.language;
-  if (locale === 'en-US') {
-    it('should return string number with locale', () => {
-      expect(formattedNumber(50000.05)).toBe(results[locale].expectNumber);
-      expect(formattedNumber('50000.05')).toBe(results[locale].expectNumber);
-      expect(formattedNumber('50000.05', 4)).toBe(
-        results[locale].expectNumberWithAccuracy
-      );
-    });
-
-    it('should return string date with locale', () => {
-      expect(formattedDate(date)).toBe(results[locale].expectDate);
-    });
-  }
+  it('should return string number with locale', () => {
```

Comment:
plz add more tests

i see at least 4-5 cases in the task multiplied by edge values for each case, like NaN, Infinity, 0, etc